src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -1133,17 +1128,13 @@ else if (fuelEnabled
 							
 							UUID carId = car.getUniqueId();
 							
-							Location toTele = new Location(s.getWorld(), x,
+							final Location toTele = new Location(s.getWorld(), x,
 									y, z);
 							Chunk ch = toTele.getChunk();
-							if (ch.isLoaded()) {
+							if (!ch.isLoaded()) {
 								ch.load(true);
 							}
-							car.teleport(toTele);
-							UEntityMeta.setMetadata(car, "carhealth", health);
-							if (raceCar) {
-								UEntityMeta.setMetadata(car, "kart.racing", new StatValue(null, plugin));
-							}
+							player.teleport(toTele.clone().add(0,2,0));
```

Comment:
Why add 2? Why not just type in the location on the sign to be at the correct y?

Reply:
Oh I see it's for the player. Probably alright then

Reply:
Yeah I'll change it to +1 tho - it made sense to me as I need to tp the player into the proximity of the cart but don't want them to collide mid-air.
+2 seems a little high in retrospect, gotta mind potential ceilings.

Reply:
Sounds good